<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminMenu</class>
 <widget class="QMainWindow" name="AdminMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#AdminMenu {
    background-color: #f4f4db;  
}

/* Загальний стиль для всіх карток */
QFrame#booksCard, 
QFrame#authorsCard, 
QFrame#categoriesCard,
QFrame#publishersCard,
QFrame#bookIssueCard,
QFrame#bookReturnedCard,
QFrame#bookNotReturnCard {
    border-radius: 8px;
    background-color: #e74c3c;   /* червоний як у прикладі */
}

/* Верхній заголовок */
QFrame#booksCard QLabel:first-child,
QFrame#authorsCard QLabel:first-child,
QFrame#categoriesCard QLabel:first-child,
QFrame#publishersCard QLabel:first-child,
QFrame#bookIssueCard QLabel:first-child,
QFrame#bookReturnedCard QLabel:first-child,
QFrame#bookNotReturnCard QLabel:first-child {
    background: rgba(0,0,0,0.15);
    padding: 6px 10px;
    font-size: 18px;
    font-weight: bold;
    color: white;
    border-top-left-radius: 8px;
    border-top-right-radius: 8px;
}

/* Основна цифра */
QFrame QLabel {
    font-size: 20px;
    font-weight: bold;
    color: white;
}

QFrame#booksCard {
    background-color: #8fb996;   /* red */
}
QFrame#authorsCard {
    background-color: #8fb996;   /* blue */
}
QFrame#membersCard,
QFrame#bookIssueCard {
    background-color: #8fb996;   /* orange */
}
QFrame#categoriesCard {
    background-color: #8fb996;   /* purple */
}
QFrame#publishersCard {
    background-color: #40916c;   /* green */
}
QFrame#bookReturnedCard {
    background-color: #8fb996;   /* light green */
}
QFrame#bookNotReturnCard {
    background-color: #8fb996;   /* dark red */
}

QMenuBar {
 background-color: #a9c3a6;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="booksCard">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="labelBooksCount">
        <property name="text">
         <string>labelBooksCount</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Total book</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="authorsCard">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="labelAuthorsCount">
        <property name="text">
         <string>labelAuthorsCount</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Total author</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="categoriesCard">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="labelCategoriesCount">
        <property name="text">
         <string>labelCategoriesCount</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Total category</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="publishersCard">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="labelPublishersCount">
        <property name="text">
         <string>labelPublishersCount</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Total publisher</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="bookIssueCard">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="labelBookIssueCount">
        <property name="text">
         <string>labelBookIssueCount</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Total book issue</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="bookReturnedCard">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>241</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="labelBookReturnedCount">
        <property name="text">
         <string>labelBookReturnedCount</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Total book returned</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="bookNotReturnCard">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>252</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="labelBookNotReturnCount">
        <property name="text">
         <string>labelBookNotReturnCount</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Total book not return</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBooks">
    <property name="title">
     <string>Books</string>
    </property>
    <addaction name="actionAdd_new_book"/>
    <addaction name="actionView_books"/>
    <addaction name="actionIssue_books"/>
    <addaction name="actionReturn_books"/>
   </widget>
   <widget class="QMenu" name="menuStudent">
    <property name="title">
     <string>Student</string>
    </property>
    <addaction name="actionAdd_student"/>
    <addaction name="actionView_student_information"/>
    <addaction name="actionEdit_student_information"/>
   </widget>
   <widget class="QMenu" name="menuUser">
    <property name="title">
     <string>User</string>
    </property>
    <addaction name="actionLogout"/>
   </widget>
   <addaction name="menuBooks"/>
   <addaction name="menuStudent"/>
   <addaction name="menuUser"/>
  </widget>
  <action name="actionAdd_new_book">
   <property name="text">
    <string>Add new book</string>
   </property>
  </action>
  <action name="actionView_books">
   <property name="text">
    <string>View books</string>
   </property>
  </action>
  <action name="actionAdd_student">
   <property name="text">
    <string>Add student</string>
   </property>
  </action>
  <action name="actionView_student_information">
   <property name="text">
    <string>View student information</string>
   </property>
  </action>
  <action name="actionComplete_book_detail">
   <property name="text">
    <string>Complete book detail</string>
   </property>
  </action>
  <action name="actionIssue_books">
   <property name="text">
    <string>Issue books</string>
   </property>
  </action>
  <action name="actionReturn_books">
   <property name="text">
    <string>Return books</string>
   </property>
  </action>
  <action name="actionEdit_student_information">
   <property name="text">
    <string>Edit student information</string>
   </property>
  </action>
  <action name="actionView_all_members">
   <property name="text">
    <string>View all members</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
